backpack/__init__.py

```diff
@@ -43,6 +44,15 @@ def __init__(self, *exts: BackpropExtension, debug=False):
         Attributes:
             args: [BackpropExtension]
                 The extensions to activate for the backward pass.
+            extension_hook: Callable, optional (default: None)
+                Function with signature ``torch.nn.Module → None``, that will be called
+                on each module after all BackPACK extensions have been computed. It
+                can be used to reduce memory overhead if the goal is to compute
+                transformations of BackPACK quantities. Information can be compacted
+                during a backward pass and obsolete tensors be freed manually.
+
+                If the callable iterates over the module parameters, it may iterate
+                multiple times over some since the hook acts on the modular level.
```

Comment:
```suggestion
            extension_hook: Callable, optional (default: None)
                Function called on each module after all BackPACK extensions have run. 
                Takes a ``torch.nn.Module`` and returns ``None``. 
                
                Can be used to reduce memory overhead if the goal is to compute
                transformations of BackPACK quantities. Information can be compacted
                during a backward pass and obsolete tensors be freed manually (``del``).

                Note: 
                    If the callable iterates over the module parameters, it may iterate
                    multiple times over some since the hook acts on the modular level.
```

Reply:
I don't think I understand the note 
```
If the callable iterates over the module parameters, it may iterate
multiple times over some since the hook acts on the modular level.
```
Wouldn't the standard backpack hook be called only once per module, and this hook as well? 
If so, it would iterate over the module parameter only once? Unless you meant `model` parameter?